Tutorials/whatiswot/3-JSON/3-JSON.txt

```diff
@@ -1,51 +1,51 @@
-In this video, we will take a closer look at JSON. Since it is the most used payload format in the Web of Things , as well as the way Thing Descriptions are represented. 
+In this video, we will take a closer look at JSON. Since it is the most used payload format in the Web of Things , as well as the way Thing Descriptions are represented.  
 
-So, let’s start with what is JSON. 
+So, let’s start with what is JSON.  
 
-JSON is a lightweight data-interchange format. 
+JSON is a lightweight data-interchange format.  
 
-It is easy for humans to read and write 
+It is easy for humans to read and write  
 
-and easy for machines to parse and generate. 
+and easy for machines to parse and generate.  
 
-Let’s take a look at how parsing and generating happens. We can parse a JSON to visualize the data such as temperature. 
+Let’s take a look at how parsing and generating happens. We can parse a JSON to visualize the data such as temperature.  
 
-Or we can generate JSON from a given data, and send it in JSON format. 
+Or we can generate JSON from a given data,  and send it in JSON format.  
 
-Let’s look at the quick facts about the JSON.  
+Let’s look at the quick facts about the JSON.   
 
 It was standardized by ECMA and  IETF in around 2013, the current version is 2017 [RFC 8259]
 
-It is used for data exchange but also for configuration files and It can be translated to other popular formats in both ways like XML,  YAML, and other formats.
+It is used for data exchange but also for configuration files and It can be translated to other popular formats in both ways like XML,   YAML, and other formats.
 
 Even though JSON stands for Java Script Object Notation, it does not only consist of objects but also allows primitive types.
 
 The confusion comes from the very early versions of JSON which only included objects and would not allow the primitive types to exist on their own.
 
 However, now JSON accepts objects and it has 5 primitive types that are also accepted on their own.
 
-Let's look closer to these primitive types; 
+Let's look closer to these primitive types;  
 
-String,  Number,  Boolean,   and null. 
+String, Number , Integer, Boolean, and null. 
 
-Now, we will talk about the structured types, which are split into two in JSON. 
+Now, we will talk about the structured types, which are split into two in JSON.  
 
-First one is name value pairs, called objects. 
+First one is name value pairs, called objects.  
 In various programming languages, this can be called an object, struct, dictionary, hash table or keyed list.
 
-Second one is ordered list of values, called arrays.  In most programming languages, this can be called as an array, vector, list, or sequence.  An array of different types is allowed too. 
+Second one is ordered list of values, called arrays.   In most programming languages, this can be called as an array, vector, list, or sequence.   An array of different types is allowed too.  
```

Comment:
As with PR #164, many odd space characters added by PR #165. This line adds three -- two in midline multi-space strings which should be single-space (`arrays.   In` and `sequence.   An`), and one before EOL (`too. `). Several lines now have 2 or 3 space chars (where there should be zero) before EOL. These errors make me wonder whether (at least some of) this PR represents regression of previously merged PR...?